README.md

```diff
@@ -26,6 +26,18 @@ If you've installed Antora globally using Yarn, you may need to add `"$(yarn glo
 
 Please read the [documentation](https://github.com/Mogztter/antora-lunr#enable-the-search-component-in-the-ui) to enable the search component in your documentation UI.
 
+### Using the Docker image
+
+[The docker image](https://hub.docker.com/r/adito5393/antora-v2.3.4-lunr) is build upon [the Antora's official docker image](https://docs.antora.org/antora/2.3/antora-container/#run-the-antora-image) and should be used as a replacement. For example, instead of running:
```

Comment:
```suggestion
[The Docker image](https://hub.docker.com/r/adito5393/antora-v2.3.4-lunr) is built upon [the Antora's official Docker image](https://docs.antora.org/antora/2.3/antora-container/#run-the-antora-image) and should be used as a replacement. For example, instead of running:
```